app/api/bucketlist_api.py

```diff
@@ -0,0 +1,67 @@
+from flask import g, request, jsonify, abort
+from app import app, db
+from app.auth.auth import auth
+from flask_login import login_required
+from app.models import Bucketlist
+
+@app.route('/bucketlists/', methods = ['GET'])
+@login_required
+def get_all_bucketlists(self):
+    """gets all the bucket lists"""
+    bucket = Bucketlist.query.filter_by(created_by=g.user_id)
+    if not bucket or bucket.created_by != g.user_id:
+        return jsonify(message='There is No Bucketlist to display!!!!')
```

Comment:
A status code of `404` should be returned here since no Bucketlist was found, don't you think??